src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -403,23 +403,22 @@ private void sendNotifications() {
         );
         // FIXME: Change log level
         LOG.info("Sending notification to user {}", trip.userId);
-        boolean success = false;
-        // FIXME: This needs to be an enum.
-        switch (otpUser.notificationChannel.toLowerCase()) {
-            case "sms":
-                success = sendSMS(otpUser, templateData);
-                break;
-            case "email":
-                success = sendEmail(otpUser, templateData);
-                break;
-            case "all":
-                // TODO better handle below when one of the following fails
-                success = sendSMS(otpUser, templateData) && sendEmail(otpUser, templateData);
-                break;
-            default:
-                break;
+        boolean successEmail = false;
+        boolean successPush = false;
+        boolean successSms = false;
+
```

Comment:
I would poll the registered devices at this point regardless of whether "push" is a selected channel:
```
           ...
            otpUser.pushDevices = getPushInfo(toUser);
            Persistence.otpUsers.replace(otpUser.id, otpUser);
```

Reply:
Okay, that'll work.

Reply:
My bad, push devices should be updated at the very top of the `sendNotifications` method, even if no notifications will be sent (otherwise the web UI will report an incorrect number of devices).

Reply:
This is changed.

Reply:
@binh-dam-ibigroup - Clarification, should I just add these same lines (call to `getPushInfo()` and then `replace()`) to the top of `sendNotifications()`?

Reply:
Updated. I did it near the very top (right after retrieving `otpUser`).